### PR TITLE
Updating code of conduct

### PR DESCRIPTION
**Pull request by woody-apple**
Updating our code of conduct, and adding contact information.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,82 +2,140 @@
 
 This Project CHIP Open Source Code of Conduct applies to all those contributing
 to, participating in, or maintaining the Project CHIP open source project,
-including Zigbee Alliance members and non-members.
+including CSA members and non-members.
```
Comment (woody-apple) [inline CODE_OF_CONDUCT.md:5]:
```suggestion
including Connectivity Standards Alliance members and non-members.
```

**Review by chrisdecenzo — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -2,82 +2,140 @@
...
-Examples of unacceptable behavior by participants include:
-
--   The use of violent threats, abusive, discriminatory, or derogatory language
--   The use of sexualized language or imagery and unwelcome sexual attention or
-    advances
--   Trolling, insulting/derogatory comments, and personal or political attacks
--   Public or private harassment
--   Publishing others' private information, such as a physical or electronic
-    address, without explicit permission
--   Posting of violent content
```
[the remark this answers is not in the document]
Comment (woody-apple) [reply CODE_OF_CONDUCT.md:36]: Most of these are still present, and this is a copy paste of the current copy here: https://www.contributor-covenant.org/version/2/1/code_of_conduct/

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -2,82 +2,140 @@
...
--   Engaging in unwanted physical contact
--   Other conduct which could reasonably be considered inappropriate in a
-    professional setting
--   Advocating for or encouraging any of the above behaviors
```
[the remark this answers is not in the document]
Comment (woody-apple) [reply CODE_OF_CONDUCT.md:40]: Most of these are still present, and this is a copy paste of the current copy here: [contributor-covenant.org/version/2/1/code_of_conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/)

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -2,82 +2,140 @@
...
+
+* The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+* Disclosing private information, unreleased products, or unreleased specification information
```
Comment (woody-apple) [inline CODE_OF_CONDUCT.md:46]:
```suggestion
* Disclosing private information, unreleased product information, or unreleased specification information
```

**Review by smides-nest — APPROVED** [review]
LGTM

**Review by leorozendaal — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -2,82 +2,142 @@
...
-    address, without explicit permission
--   Posting of violent content
--   Engaging in unwanted physical contact
+-   Publishing others' private information, such as a physical or email address,
+    without their explicit permission
 -   Other conduct which could reasonably be considered inappropriate in a
     professional setting
--   Advocating for or encouraging any of the above behaviors
+-   Disclosing private information, unreleased products
+-   Disclosing of unreleased specification information outside of Connectivity
```
Comment (woody-apple) [inline CODE_OF_CONDUCT.md:47]:
```suggestion
-   Disclosure of private information, unreleased products
-   Disclosure of unreleased specification information outside of Connectivity
```

**Pull request merged**